packages/neuron-ui/src/components/NervosDAODetail/CellsCard/cellsCard.module.scss

```diff
@@ -92,7 +89,32 @@
       .address {
         font-family: 'JetBrains Mono';
         font-weight: 500;
+        display: inline-block;
+        line-height: 56px;
       }
+
+      .balance {
+        height: 32px;
+        line-height: 32px;
+      }
+
+      .scriptTag:hover {
+        color: var(--primary-color);
+      }
+    }
+  }
+
+  .copyTableAddress {
+    word-break: break-all;
+    text-align: center;
+    font-family: 'JetBrains Mono';
```

Comment:
The font family appears multiple times with its literal value, maybe using a mixin helps.